iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/TransportClientTests.java

```diff
@@ -210,21 +210,16 @@ public static void tearDown() throws IOException
         }
     }
 
-    @Before
-    public void setupTestInstance() throws InterruptedException, IotHubException, URISyntaxException, IOException, GeneralSecurityException
-    {
-        testInstance.setup();
```

Comment:
Moving this to a per-test basis since at least one test in this suite requires a few test identities with no twin, while the rest are fine with using a modified twin.